docs/reference/ml/df-analytics/apis/analysisobjects.asciidoc

```diff
@@ -145,7 +145,8 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=lambda]
 include::{docdir}/ml/ml-shared.asciidoc[tag=dependent-variable]
 +
 --
-The data type of the field must be numeric or boolean.
+The data type of the field must be numeric (`integer`, `short`, `byte`) or 
```

Comment:
Please extend this list.
Dependent variable of classification analysis must be one of:
- text, keyword or ip
- byte, short, integer or long
- boolean